docs/Assets.md

```diff
@@ -0,0 +1,105 @@
+# Assets and JSON module imports
+
+UI5 Web Components aim to be feature rich and with a minimal code footprint at the same time. In order to achieve this, 
+most UI5 Web Components packages ship their assets as `.json` files while also providing respective public module imports for them.
+
+The assets in question could be i18n texts, icons, additional themes parameters, CLDR, etc...
+
+Currently our npm packages follow the scheme:
+
+`@ui5/<PACKAGE_NAME>/dist/assets/*`
+(for the assets themselves)
+
+`@ui5/<PACKAGE_NAME>/dist/json-imports/*`
+(for the modules that provide the assets)
+
+<a name="packages"></a>
+## Packages
+
+### `base` package
+
+Asset type | Asset path | Public Module Import
+------|------------------|---
+CLDR | `@ui5/webcomponents-base/dist/assets/cldr/*` | `@ui5/webcomponents-base/dist/json-imports/LocaleData.js`
+
+Usually you don't need to import the CLDR assets from the `base` package, as the other packages provide handy shortcuts for the purpose.
```

Comment:
remove shortcut, it is exactly the same in other packages

Reply:
done